src/options/prop_options.toml

```diff
@@ -1,6 +1,19 @@
 id     = "PROP"
 name   = "SAT Layer"
 
+[[option]]
+  name       = "satSolver"
+  category   = "expert"
+  long       = "sat-solver=MODE"
+  type       = "SatSolverMode"
+  default    = "MINISAT"
```

Comment:
Why all caps here?

Reply:
This is the name of the corresponding enum value, e.g., `SatSolverMode::MINISAT`. That's why it has to be upper case.

Reply:
Got it.